SECONDARYINDEX

```diff
@@ -1,24 +0,0 @@
-welcome:
-getting-started:tutorial/0-getting-started
-1-basic-predicate-calculus:tutorial/1-basic-predicate-calculus
-what-is-coq:about-coq
-adt:adt-coq
-coq-82-detailed-description:coq-82
-journée-de-démarrage-de-ladt:adt/demarrage
-journée-«-bibliothèques-»:adt/biblios
-journée-«-bibliothèques-»-du-11-décembre-2008:adt/biblios
-journée-«-automatisation-»-du-24-mars-2009:adt/automatisation
-the-coq-workshop-2009-0:coq-workshop/2009
-journée-«-tactiques-»-du-30-juin-2009:adt/tactiques
-the-coq-workshop-2010:coq-workshop/2010/cfp
-the-coq-workshop-2010-0:coq-workshop/2010
-the-coq-workshop:coq-workshop
-journée-«-égalité-et-terminaison-»-du-2-février-2010:adt/egalite-terminaison
-journée-«-interfaces-»-du-27-octobre-2010:adt/interfaces
-the-coq-workshop-2012:coq-workshop-2012
-coq-84-0:coq-8.4
-coq-83:coq-8.3
```

Comment:
Check that the remaining URLs are all consistent: we have https://coq.inria.fr/coq-87, https://coq.inria.fr/coq-86, https://coq.inria.fr/coq-85.

Reply:
I’m not sure I understand this comment. Do you mean something more than the suggestion below? Thanks.

Reply:
Nothing more. But you can delete this page as well I think (https://coq.inria.fr/coq-8.4).

Reply:
Well spotted. Done. Thanks.